### DISCUSSION TITLE
Generate New Wallet - Add a confirmation page

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#2274] [Ideas / Feature Requests]
Right now, when user generates a **new wallet,** they're presented with a `recovery-seed-backup` page which shows the recovery seed. However, for someone (not good in English, or drunk) it can look like a success page which they quickly skip through which will lead to a _user not backing up the recovery seed_. 

![Screenshot from 2021-02-11 13-05-35](https://user-images.githubusercontent.com/36959754/107634740-4da62780-6c6a-11eb-8a59-6c2c706c0407.png)

Now, if the generated wallet is hot wallet (private key held on a server) that's fine because at some point they'll be able to spend funds. But, for watch-only wallet this may completely lock their funds.

We should invest reasonable effort to either add a `are you sure page` and explain what's being done again, just to make sure a user doesn't skip through this essential step. This is an example of such page:

![Screenshot from 2021-02-11 13-12-01](https://user-images.githubusercontent.com/36959754/107635020-c1483480-6c6a-11eb-8a3a-8cb182c15775.png)

Alternatively we can add seed confirmation page, but perhaps this one is quicker fix.

What do you guys think?

**Comment by dstrukt** [comment]
Completely agree, and currently in progress!

**Reply by bolatovumar** [reply]
Also agree with this. We can't be too careful with anything that involves users potentially losing access to their funds. We should make the design as fool-proof as possible.
